spec/requests/user_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+RSpec.describe 'Users', type: :request do
+  describe 'Get and checks User ' do
+    before(:example) do
+      get '/users'
+    end
+
+    it 'Checks if the status response is succes(OK)' do
+      expect(response.status).to be(200)
+    end
+
+    it 'Checks if it render the correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Check the placeholder text' do
+      expect(response.body).to include('<p>this is a list of users</p>')
+    end
+  end
+
+  describe 'Check specific user' do
+    before(:example) do
+      get '/users/1'
+    end
+
+    it 'Checks if the status response is succes(OK)' do
+      expect(response.status).to be(200)
+    end
+
+    it 'Checks if it render the correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'Check if it has the correct placeholder text' do
+      expect(response.body).to include('<p>This is a specified user</p>')
+    end
+  end
+end
```

Comment:
- [ ] Some of your tests are failing. Kindly fix them.

  ![asew](https://user-images.githubusercontent.com/54848363/198273956-1eda7cc3-4e73-4ea1-86c3-0493126e9d9c.jpg)

Reply:
Thanks for your observation @DammyShittu , everything is currently passing on my local computer 🤔 kindly
![Screenshot (152)](https://user-images.githubusercontent.com/64643995/198276676-38890b98-b0a4-4a5a-9d6c-9387208c8b3c.png)
 check the picture attached. I ran the test severally and they are passing.